go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/kubev2v/migration-planner
 
-go 1.22.2
+go 1.23.6
+
+toolchain go1.24.0
```

Comment:
Can you please write a comment why we need 1.24 toolchain, so other knows

Reply:
Done